rcl/test/rcl/test_remap.cpp

```diff
@@ -19,9 +19,9 @@
 #include "rcl/remap.h"
 #include "rcl/error_handling.h"
 
+#include "./allocator_testing_utils.h"
 #include "./arg_macros.hpp"
 #include "./arguments_impl.h"
-#include "./allocator_testing_utils.h"
 #include "./remap_impl.h"
```

Comment:
how does `./remap_impl.h` works if the header is not in the same folder?

Reply:
I see https://github.com/ros2/rcl/blob/70eba5e788af95c0e9f0dfe71af9ae630a9d4d76/rcl/test/CMakeLists.txt#L182
In this case, it's better to use `remap_impl.h` without `./`.
`./` should be reserved for includes that are actually in the same directory.

Reply:
`cpplint` complains if I don't include the directory for `.h` files